#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_remove_old_reference_to_menu_h.dpatch by  <goedson@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removes a reference to a non-existing menu.h file

@DPATCH@
Index: b/src/menu.c
===================================================================
--- a/src/menu.c	2009-12-19 17:11:48.000000000 -0200
+++ b/src/menu.c	2009-12-19 17:12:10.000000000 -0200
@@ -34,7 +34,6 @@
 #include "image.h"
 #include "image_processing.h"
 #include "tool_palette.h"
-#include "menu.h"
 #include "print.h"
 #include "file.h"
 #include "text.h"
